packages/formstr-app/src/containers/ResponsesNew/Export.tsx

```diff
@@ -7,17 +7,30 @@ export const Export: React.FC<{
   responsesData: Array<{ [key: string]: string }>;
   formName: string;
 }> = ({ responsesData, formName }) => {
+  const hasResponses = responsesData.length > 0;
+
   const onDownloadClick = async (type: "csv" | "excel") => {
+    if (!hasResponses) {
+      alert("No responses to export");
+      return;
+    }
+
+   try {
```

Comment:
generally try blocks should be concise and only encapsulate code, you are expecting to throw errors